packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -762,55 +762,76 @@ export const guides = [
-          },
-          {
-            title: "Enable Terraform access",
-            guideName: "terraform-enable-org-access",
-          },
-          {
-            title: "Manage user access",
-            guideName: "terraform-manage-user-access",
-          },
-          {
-            title: "Manage groups",
-            guideName: "terraform-manage-groups",
-          },
-          {
-            title: "Manage authentication services",
-            guideName: "terraform-manage-external-authenticators",
-          },
-          {
-            title: "Manage device connection requirements",
-            guideName: "terraform-configure-device-signin-standards"
-          },
-          {
-            title: "Manage authorization servers",
-            guideName: "terraform-create-custom-auth-server"
-          },
-          {
-            title: "Customize end-user experience",
-            guideName: "terraform-manage-end-user-experience",
-          },
-          {
-            title: "Control Terraform access",
-            guideName: "terraform-design-access-security",
-          },
-          {
-            title: "Optimize Terraform access",
-            guideName: "terraform-design-rate-limits"
+            title: "Essentials",
+            subLinks: [
+              {
+                title: "Terraform overview",
+                guideName: "terraform-overview",
+              },
+              {
+                title: "Enable Terraform access",
+                guideName: "terraform-enable-org-access",
+              },
+              {
+                title: "Control Terraform access",
+                guideName: "terraform-design-access-security",
+              },
+            ],
           },
           {
-            title: "Organize your configuration",
-            guideName: "terraform-organize-configuration"
+            title: "Create and manage resources",
+            subLinks: [
+              {
+                title: "Manage groups",
+                guideName: "terraform-manage-groups",
+              },
+              {
+                title: "Manage user access",
+                guideName: "terraform-manage-user-access",
+              },
+              {
+                title: "Manage authentication services",
+                guideName: "terraform-manage-external-authenticators",
+              },
+              {
+                title: "Manage device requirements",
+                guideName: "terraform-configure-device-signin-standards"
+              },
+              {
+                title: "Manage authorization servers",
+                guideName: "terraform-create-custom-auth-server"
+              },
+              {
+                title: "Manage branding",
+                guideName: "terraform-manage-end-user-experience",
+              },
+              {
+                title: "Manage custom domains",
+                guideName: "terraform-mange-multiple-domains"
```

Comment:
Just flagging the mange here in case you decide to fix it.

Reply:
Fixed in next commit